cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/rolling/RackRolling.java

```diff
@@ -223,19 +278,15 @@ static List<Set<KafkaNode>> cells(Reconciliation reconciliation,
 
     /**
      * Pick the "best" batch to be restarted.
-     * This is the largest batch of available servers excluding the
+     * This is the largest batch of available servers
      * @return the "best" batch to be restarted
      */
-    static Set<KafkaNode> pickBestBatchForRestart(List<Set<KafkaNode>> batches, int controllerId) {
+    static Set<KafkaNode> pickBestBatchForRestart(List<Set<KafkaNode>> batches) {
```

Comment:
Why did you change this?

I don't recall now if the original idea was to always put the controller in its own batch, or just make the batch with the controller be the last batch.

Reply:
We were passing the active controlIer ID to this function and placing the batch at the end. However we put the active controller broker into `BROKER_AND_ACTIVE_CONTROLLER` group and return a separate batch for it. So I think the batches passed into this function never actually contained the active controller. 

Otherwise, we could remove `BROKER_AND_ACTIVE_CONTROLLER` group and the keep this logic to start the batch with active controller at the end. I'm happy with either.